app/assets/stylesheets/application.scss

```diff
@@ -55,6 +55,7 @@ html, body {
 body {
   display: flex;
   flex-direction: column;
+  margin-top: -50px;
```

Comment:
This has the effect that the body is ending 50px short of the bottom of the viewport. Thus the footer is not displayed at the bottom of the page but 50px above if you look at a page with little content.

Reply:
Please do some manual testing with dev and your branch side-by-side!